src/librustc_trans/save/dump_csv.rs

```diff
@@ -653,8 +647,8 @@ impl <'l, 'tcx> DumpCsvVisitor<'l, 'tcx> {
         }
     }
 
-    fn process_mod(&mut self,
-                   item: &ast::Item) {  // The module in question, represented as an item.
+    // `item` is the module in question, represented as an item.
```

Comment:
This was actually a manual fixup, so intentional. The comment is on the arg, so it doesn't matter if it is inside or outside the function.

(This is actually due to an interesting bug in rustfmt - we check to see if everything will fit on one line, then deal with 'inner' comments, in this case by appending them, which causes the line to overflow. It's tricky because the only indicator that the comment will start on the same line is the absence of a newline which isn't in anything's span. So I had to patch around the bug for now...).